packages/atlas/src/components/_overlays/SendTransferDialogs/SendFundsDialog.tsx

```diff
@@ -74,7 +84,25 @@ export const SendFundsDialog: FC<SendFundsDialogProps> = ({ onExitClick, account
   )
 
   const handleSendFounds = async () => {
-    const handler = await handleSubmit(() => null)
+    const handler = await handleSubmit((data) => {
+      if (!joystream || !data.account || !data.amount) {
+        return
+      }
+      handleTransaction({
+        snackbarSuccessMessage: {
+          title: `${formatNumber(
+            data.amount
+          )} ${JOY_CURRENCY_TICKER} (${convertedAmount}) tokens have been sent over to ${data.account.slice(
+            0,
+            ADDRESS_CHARACTERS_LIMIT
+          )}...
+          ${data.account.slice(-ADDRESS_CHARACTERS_LIMIT)} wallet address`,
+        },
+        txFactory: async (updateStatus) =>
+          (await joystream.extrinsics).sendFunds(data.account || '', amount, proxyCallback(updateStatus)),
```

Comment:
we should use `data.amount` here. Also, this should be multiplied by 10_000_000_000. We're sending tokens and each token is `10_000_000_000` hapi.